docs/source/overview/delivery_services.rst

```diff
@@ -0,0 +1,755 @@
+.. tip:: The only way to get the integral, unique identifier of a :term:`Type` of Delivery Service is to look at the database after it has been generated; these are non-deterministic and cannot be guaranteed to have any particular value, or even consistent values. This can be done directly or, preferably, using the :ref:`to-api-types` endpoint. Unfortunately, knowing the name of the :term:`Type` is rarely enough for many applications. The ``useInColumn`` values of these :term:`Types` will be ``deliveryservice``.
+
+DNS
+	Delivery Services of this routing type are routed by Traffic Router by providing DNS records that provide the IP addresses of :term:`cache servers` when clients look up the full Delivery Service :abbr:`FQDN (Fully Qualified Domain Name)`.
+HTTP
+	The Traffic Router(s) responsible for routing this Delivery Service will still answer DNS requests for the Delivery Service :abbr:`FQDN (Fully Qualified Domain Name)`, but will provide its own IP address. The client then directs its HTTP request to the Traffic Router, which will use an `HTTP redirection response <https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#Redirection_messages>`_ to direct the client to a :term:`cache server`.
+
+More generally, though, Delivery Services have a Type that defines not only how traffic is routed, but also how content is cached and semantically defines what "content" means in the context of a given Delivery Service.
+
+ANY_MAP
+	This is a special kind of Delivery Service that should only be used when control over the clients is guaranteed, and very fine control over the :abbr:`ATS (Apache Traffic Server)` `remap.config  <https://docs.trafficserver.apache.org/en/7.1.x/admin-guide/files/remap.config.en.html>`_ line for this Delivery Service is required. ANY_MAP is not known to Traffic Router. It is not routed in any way. For Delivery Services of this type, the "Raw Remap Text" field **must** be defined, as it is the only configuration generated by Traffic Control. The only way for a client to utilize delivery through an ANY_MAP service is by knowing in advance the IP address of one or more :term:`Edge-tier cache server`\ s and make the appropriate request(s).
+DNS
+	Uses DNS content routing. Delivers content normally. This is the recommended Type for delivering smaller objects like web page images.
+DNS_LIVE\ [#dupOrigin]_
+	Uses DNS Content routing, but optimizes caching for live video streaming. Specifically, the configuration generated for :term:`cache servers` responsible for serving content for this Delivery Service will not cache that content on storage disks. Instead, they will make use of RAM block devices dedicated to ATS - as specified by the special ``RAM_Drive_Prefix`` and ``RAM_Drive_Letters`` :term:`Parameters`. Also, any :term:`Mid-tier` of caching is bypassed.
+DNS_LIVE_NATNL
+	Works exactly the same as DNS_LIVE, but is optimized for delivery of live video content across a wide physical area. What this means is that the :term:`Mi
```

Comment:
TR doesn't really "eliminate all but the closest location to the client as possibilities." It still includes other locations as possibilities, ordered by proximity to the client's location. This is the same for STEERING_GEO_WEIGHT.

Reply:
Uh, I think you put that on the wrong line. You're talking about STEERING_GEO_ORDER, right?

Reply:
Yeah, not sure what happened there.

Reply:
how's it look now?